src/current/_includes/v24.3/ldr/show-logical-replication-responses.md

```diff
@@ -0,0 +1,9 @@
+Field    | Response
+---------+----------
+`job_id` | The job's ID. Use with [`CANCEL JOB`]({% link {{ page.version.version }}/cancel-job.md %}), [`PAUSE JOB`]({% link {{ page.version.version }}/pause-job.md %}), [`RESUME JOB`]({% link {{ page.version.version }}/resume-job.md %}), [`SHOW JOB`]({% link {{ page.version.version }}/show-jobs.md %}).
+`status` | Status of the job `running`, `paused`, `canceled`. {% comment  %}check these{% endcomment %}
```

Comment:
is this replicating statuses from the `SHOW JOBS` output? or is LDR job special ?

PS just ignore this question if you need to merge ASAP, it's non-blocking

Reply:
You're right, checked with eng and the show logical replication jobs is syntactic sugar for `show jobs`... so I added the same statuses here. And probably another place for an include.